lib/protoparser/prometheus/parser.go

```diff
@@ -185,25 +198,86 @@ func (r *Row) unmarshal(s string, tagsPool []Tag, noEscapes bool) ([]Tag, error)
 	return tagsPool, nil
 }
 
-var rowsReadScrape = metrics.NewCounter(`vm_protoparser_rows_read_total{type="promscrape"}`)
+var protobufRowsReadScrape = metrics.NewCounter(`vm_protoparser_rows_read_total{type="promscrape",format="PrometheusProto"}`)
+
+func unmarshalProtobuf(dst []Row, s []byte, tagsPool []Tag, errLogger func(s string)) ([]Row, []Tag) {
```

Comment:
Looks like `errLogger` is not needed for this function.

Reply:
will fix all lint issues later, first want to understand if in general I'm moving towards a right direction